frontends/web/src/locales/en/app.json

```diff
@@ -738,6 +738,10 @@
         "text": "For the BitBox01, click on the BitBox icon in the sidebar on the left and see the Pairing section. The guide will update and you can continue following the instructions from there.\nFor the BitBox02, you can verify addresses directly on the device during the send/receive process.",
         "title": "How can I verify an address securely?"
       },
+      "plugout": {
+        "text": "No, once you send coins to your BitBox address, you do not need to leave your BitBox plugged in. This is because the transaction has already been broadcasted to the blockchain and is on its way to your BitBox address. You are free to disconnect your BitBox.",
```

Comment:
'once you sent'?

broadcasted -> broadcast (past tense of broadcast is broadcast)

I'd actually drop the whole sentence 'This is because the transaction has already been broadcasted to the blockchain and is on its way to your BitBox address.' as it is quite inaccurate :shrug:

Reply:
I think it's nice to have a high level sentence explaining why they don't need to have it plugged in. I'm fine to remove for now as not to block the release. But I'll try to think way to explain it more accurately without being too technical.

Reply:
:+1: can always update it on locize later

Reply:
In the future a not too technical high level sentence explaining why would be nice.

fixed in 
https://github.com/digitalbitbox/bitbox-wallet-app/compare/a9cc9deb8bca1a52da3b0843c9677ed8488529f2..a568f6a533009adc0997c43348819fc82f0fbb00